Jenkinsfile

```diff
@@ -0,0 +1,54 @@
+pipeline {
+    agent none
+    environment {
+        projectName = 'DDF'
+    }
+    stages {
+        stage('Setup') {
+            steps{
+                notifySlack('good', "STARTED: ${JOB_NAME} ${BUILD_NUMBER} ${BUILD_URL}")
+            }
+        }
+        stage('Parallel Build') {
+            // TODO refactor this stage from scripted syntax to declarative syntax to match the rest of the stages - https://issues.jenkins-ci.org/browse/JENKINS-41334
+            steps{
+                parallel(
+                    linux: {
+                        node('linux-large') {
+                            timeout(time: 3, unit: 'HOURS') {
+                                checkout scm
+                                withMaven(maven: 'M3', mavenSettingsConfig: 'codice-maven-settings') {
+                                    sh 'mvn clean install'
+                                }
+                            }
+                        }
+                    }, windows: {
+                        node('proxmox-windows'){
+                            timeout(time: 3, unit: 'HOURS') {
+                                checkout scm
+                                withMaven(maven: 'M3', mavenSettingsConfig: 'codice-maven-settings') {
```

Comment:
we should add a global settings file called something generic like `default-global-settings` that contains stuff like any maven mirrors and other things specific to the build environment since the Maven Pipeline Plugin doesn't honor the default settings files provided by jenkins master until [JENKINS-39407](https://issues.jenkins-ci.org/browse/JENKINS-39407) is resolved